requirements-testing.txt

```diff
@@ -0,0 +1,6 @@
+boto3
+black[jupyter]
+plotly
+pyqtgraph
+git+https://github.com/ConservationInternational/trends.earth-schemas.git@v2.1.2
+git+https://github.com/ConservationInternational/trends.earth-algorithms.git@v2.1.2
```

Comment:
Same comment as above - need to use the plugin_setup invoke task to pull in the plugin dependencies to ensure that the CI sever isn't using a different setup from that shipped with the plugin.